content/v2.0/get-started.md

```diff
@@ -88,6 +88,109 @@ References to the InfluxDB user interface (UI) or localhost:9999 refer to your
 {{< cloud-name >}} UI.
 {{% /cloud %}}
 
+### (Optional) Download and install the influx CLI
+
+You can write, query, and process data through the Cloud user interface (UI) or  `influx` CLI. To download and install the `influx` CLI, complete the following steps:
```

Comment:
To use the `influx` CLI to manage and interact with your InfluxDB Cloud instance, complete the following steps: